### PR TITLE
Fix EE version for OAuth config settings

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2404,8 +2403,8 @@ Choose whether OAuth can be used for account creation and login. Options include
 
     - **Do not allow sign-in via an OAuth 2.0 provider**
     - **GitLab** (see `GitLab Settings <http://docs.mattermost.com/administration/config-settings.html#id14>`__ for more detail)
```
Comment (amyblais) [inline source/administration/config-settings.rst]:
```suggestion
    - **GitLab** (see `GitLab Settings <https://docs.mattermost.com/administration/config-settings.html#gitlab>`__ for more detail)
```

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2404,8 +2403,8 @@ Choose whether OAuth can be used for account creation and login. Options include
...
-    - **Google Apps** (see `Google Settings <http://docs.mattermost.com/administration/config-settings.html#google-enterprise>`__ for more detail)
-    - **Office 365** (see `Office 365 Settings <http://docs.mattermost.com/administration/config-settings.html#office-365-enterprise>`__ for more detail)
+    - **Google Apps** (available in Enterprise Edition E20, see `Google Settings <http://docs.mattermost.com/administration/config-settings.html#google-enterprise>`__ for more detail)
```
Comment (amyblais) [inline source/administration/config-settings.rst]:
```suggestion
    - **Google Apps** (available in Enterprise Edition E20, see `Google Settings <https://docs.mattermost.com/administration/config-settings.html#google>`__ for more detail)
```

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -2404,8 +2403,8 @@ Choose whether OAuth can be used for account creation and login. Options include
...
+    - **Office 365** (available in Enterprise Edition E20, see `Office 365 Settings <http://docs.mattermost.com/administration/config-settings.html#office-365-enterprise>`__ for more detail)
```
Comment (amyblais) [inline source/administration/config-settings.rst]:
```suggestion
    - **Office 365** (available in Enterprise Edition E20, see `Office 365 Settings <https://docs.mattermost.com/administration/config-settings.html#office-365>`__ for more detail)
```

**Comment by jasonblais** [comment]
@amyblais Thanks! Updated

File: source/administration/prev-config-settings.rst [hunk 4]
```diff
@@ -832,9 +831,9 @@ Select OAuth 2.0 service provider:
 Choose whether OAuth can be used for account creation and login. Options include:
 
     - **Do not allow sign-in via an OAuth 2.0 provider**
-    - **GitLab** (see `GitLab Settings <http://docs.mattermost.com/administration/config-settings.html#id14>`__ for more detail)
-    - **Google Apps** (see `Google Settings <http://docs.mattermost.com/administration/config-settings.html#google-enterprise>`__ for more detail)
-    - **Office 365 (Beta)** (see `Office 365 Settings <http://docs.mattermost.com/administration/config-settings.html#office-365-enterprise>`__ for more detail)
+    - **GitLab** (see `GitLab Settings <https://docs.mattermost.com/administration/config-settings.html#id14>`__ for more detail)
```
Comment (amyblais) [inline source/administration/prev-config-settings.rst]:
```suggestion
    - **GitLab** (see `GitLab Settings <https://docs.mattermost.com/administration/prev-config-settings.html#gitlab>`__ for more detail)
```

File: source/administration/prev-config-settings.rst [hunk 5]
```diff
@@ -832,9 +831,9 @@ Select OAuth 2.0 service provider:
...
+    - **Google Apps** (available in Enterprise Edition E20, see `Google Settings <https://docs.mattermost.com/administration/config-settings.html#google-enterprise>`__ for more detail)
```
Comment (amyblais) [inline source/administration/prev-config-settings.rst]:
```suggestion
    - **Google Apps** (available in Enterprise Edition E20, see `Google Settings <https://docs.mattermost.com/administration/prev-config-settings.html#google>`__ for more detail)
```

File: source/administration/prev-config-settings.rst [hunk 6]
```diff
@@ -832,9 +831,9 @@ Select OAuth 2.0 service provider:
...
+    - **Office 365 (Beta)** (available in Enterprise Edition E20, see `Office 365 Settings <https://docs.mattermost.com/administration/config-settings.html#office-365-enterprise>`__ for more detail)
```
Comment (amyblais) [inline source/administration/prev-config-settings.rst]:
```suggestion
    - **Office 365 (Beta)** (available in Enterprise Edition E20, see `Office 365 Settings <https://docs.mattermost.com/administration/prev-config-settings.html#office-365>`__ for more detail)
```

**Pull request merged**